common_package_py/commands.py

```diff
@@ -0,0 +1,344 @@
+import json
+import rclpy
+from enum import Enum
+from common_package_py.common_node import CommonNode
+
+
+class data_type(Enum):
+    """Enumeration representing the data types."""
+    NULL = 1
+    BOOLEAN = 2
+    NUMBER = 3
+    NUMBER_FLOAT = 4
+    NUMBER_INTEGER = 5
+    NUMBER_UNSIGNED = 6
+    STRING = 7
+    ARRAY = 8
+    OBJECT = 9
+
+
+class JsonKeyDefinition:
+    """
+    Represents a definition for a JSON key.
+
+    Args:
+        required (bool): Indicates whether the key is required.
+        data_types (set[data_type] | data_type): Set of data types that the key can have or a single data type.
+        min_val (float | None, optional): Minimum value allowed for numeric keys. Defaults to None.
+        max_val (float | None, optional): Maximum value allowed for numeric keys. Defaults to None.
+    """
+
+    def __init__(self, required: bool, data_types: set[data_type] | data_type, min_val: float | None = None, max_val: float | None = None):
```

Comment:
Return type & Optional
```suggestion
    def __init__(self, required: bool, data_types: Union[set[data_type], data_type], min_val: Optional[float] = None, max_val: Optiona[float] = None) -> None:
```

Reply:
Das Union wird in Python 3.10 nicht mehr benötigt.
Aus Gründen der Einheitlichkeit lasse ich `Optional` auch raus.